test/unit/spec/signaling/v2/peerconnection.js

```diff
@@ -185,6 +185,14 @@ describe('PeerConnectionV2', () => {
         encodings: [{}, {}, {}],
         preferredCodecs: { audio: [], video: [{ codec: 'vp8', simulcast: true }] }
       },
+      {
+        browser: 'chrome',
+        testName: 'does not update encodings when track is stopped',
+        readyState: 'ended',
```

Comment:
Can we update some of the positive tests to include a readyState with a value of 'live'? To make sure tests are still passing.

Reply:
when readyState is not explicitly specified, test takes it as 'live'. see line 240

Reply:
but let me change one test to explicitely pass it anyways